common/djangoapps/dark_lang/tests.py

```diff
@@ -141,10 +151,81 @@ def test_accept_mixed_case(self):
         ).save()
 
         self.assertAcceptEquals(
-            'rel-ter;q=1.0',
+            'rel-ter;q=1.0, rel-ter;q=0.5',
```

Comment:
The accept header asks for:
`rel-ter;q=1.0, rel;q=0.5`

We have `rel-ter` released, and `rel` is not but will fuzzy-match to `rel-ter`